pkg/parser/extended_syntax_parser_test.go

```diff
@@ -243,3 +244,109 @@ func TestExtendedShowTrainStmt(t *testing.T) {
 		a.Equal(11, idx)
 	}
 }
+
+func TestExtendedSyntaxParseToRun(t *testing.T) {
+	a := assert.New(t)
+	{
+		testToRun := `TO RUN a_data_scientist/ts_data_processor:1.0;`
+		r, idx, e := parseSQLFlowStmt(testToRun)
+		a.NoError(e)
+		a.True(r.Extended)
+		a.True(r.Run)
+		a.Equal(`a_data_scientist/ts_data_processor:1.0`, r.ImageName)
+		a.Equal(len(r.Parameters), 0)
+		a.Equal(len(r.OutputTables), 0)
+		a.Equal(len(testToRun), idx)
+	}
+
+	{
+		testToRun := `TO RUN a_data_scientist/ts_data_processor:1.0
+CMD "slide_window_to_row";`
+		r, idx, e := parseSQLFlowStmt(testToRun)
+		a.NoError(e)
+		a.True(r.Run)
+		a.True(reflect.DeepEqual(r.Parameters, []string{`slide_window_to_row`}))
+		a.Equal(len(r.OutputTables), 0)
+		a.Equal(len(testToRun), idx)
+	}
+
+	{
+		testToRun := `TO RUN a_data_scientist/ts_data_processor:1.0
+CMD "slide_window_to_row"
+INTO output_table;`
+		r, idx, e := parseSQLFlowStmt(testToRun)
+		a.NoError(e)
+		a.True(r.Run)
+		a.True(reflect.DeepEqual(
+			r.Parameters,
+			[]string{`slide_window_to_row`}))
+		a.True(reflect.DeepEqual(r.OutputTables, []string{`output_table`}))
+		a.Equal(len(testToRun), idx)
+	}
+
+	{
+		testToRun := `TO RUN a_data_scientist/ts_data_processor:1.0
+CMD "slide_window_to_row"
+INTO output_table_1, output_table_2;`
+		r, idx, e := parseSQLFlowStmt(testToRun)
+		a.NoError(e)
+		a.True(r.Run)
+		a.True(reflect.DeepEqual(
+			r.Parameters,
+			[]string{`slide_window_to_row`}))
+		a.True(reflect.DeepEqual(
+			r.OutputTables,
+			[]string{`output_table_1`, `output_table_2`}))
+		a.Equal(len(testToRun), idx)
+	}
+
+	{
+		testToRun := `TO RUN a_data_scientist/ts_data_processor:1.0
+CMD "slide_window_to_row", "--param_a=value_a", "--param_b=value_b"
```

Comment:
Can we omit the quotation mark?

Reply:
Good question. The strings after `CMD` is the command line arguments. There are a lot of formats/patterns for the args.
```
-argument value
--argument=value
/argument value
-short_arg
--verbose
```

What's more, the preserved keyword `SELECT`, `TO`, `INTO` can be used as the arguments too because the executable inside the docker image is developed by the users.

It will make our parser complicated. Make the command line argument as string in SQL statement, and let the executable in the docker image to parser it. It will be cleaner.

Reply:
Per discussion with @Yancey1989 , we keep the quotation mark. The reason is the same with the comment above.